tools/src/main/scala/org/apache/spark/sql/rapids/tool/qualification/QualAppInfo.scala

```diff
@@ -228,8 +255,22 @@ class QualAppInfo(
         val existingIssues = sqlIDtoProblematic.getOrElse(sqlID, Set.empty[String])
         sqlIDtoProblematic(sqlID) = existingIssues ++ issues
       }
+      // Get the write data format
+      if (node.name.contains("InsertIntoHadoopFsRelationCommand")) {
+        val writeFormat = node.desc.split(",")(2)
+        writeDataFormat += writeFormat
+      }
     }
   }
+
+  def writeFormatNotSupported(writeFormat: ArrayBuffer[String]): String = {
+    val supportedWriteDataFormat = Array("parquet", "orc")
```

Comment:
we should add write supported ops to ./tools/target/classes/supportedDataSource.csv... we don't care about the specific types but we should put formats